app/assets/js/html5lightbox/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
You should remove it. Now you have 2 jQuery in your project. this one here and another one from bower.